sqldelight-gradle-plugin/build.gradle

```diff
@@ -46,6 +46,11 @@ dependencies {
 }
 
 test {
+  inputs.files("../gradle/dependencies.gradle").withPathSensitivity(PathSensitivity.RELATIVE)
+  inputs.property("ANDROID_HOME", System.getenv("ANDROID_HOME"))
+  inputs.property("TRAVIS_OS_NAME", System.getenv("TRAVIS_OS_NAME"))
+  inputs.dir("../gradle/wrapper").withPathSensitivity(PathSensitivity.RELATIVE)
```

Comment:
Why are you using the wrapper as an input? Use [TestKit](https://docs.gradle.org/current/userguide/test_kit.html) to test plugins instead.

Reply:
We are using it in this project